packages/catalog-model/src/kinds/TemplateEntityV1alpha1.ts

```diff
@@ -33,6 +33,7 @@ export interface TemplateEntityV1alpha1 extends Entity {
     templater: string;
     path?: string;
     schema: JSONSchema;
+    owner: string;
```

Comment:
You can't add a required field like this without breaking ingestion for all template users everywhere. It has to have the optionality question mark, at least until we bump to a plain v1.

Reply:
Additionally, the same thing needs to be done on `TemplateEntityV1beta2.ts`.

Reply:
Additionally, this needs updating

- [the docs](https://github.com/backstage/backstage/blob/master/docs/features/software-catalog/descriptor-format.md#kind-template)
- [the json schemas](https://github.com/backstage/backstage/tree/master/packages/catalog-model/src/schema/kinds)
- and the new field should emit relations in the [relevant processor](https://github.com/backstage/backstage/blob/master/plugins/catalog-backend/src/ingestion/processors/BuiltinKindsEntityProcessor.ts)

all of this including updating the tests etc :) model changes are pretty important and involved ...